lib/questions.rb

```diff
@@ -56,6 +69,7 @@ def make_numbers_negative(number)
 # even numbers come first
 # so [1, 2, 3, 4, 5, 6] becomes [[2, 4, 6], [1, 3, 5]]
 def separate_array_into_even_and_odd_numbers(array)
+  array.partition{|x| x.even?}
```

Comment:
Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.<br>Pass `&:even?` as an argument to `partition` instead of a block.